satellite/satellitedb/utils.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"flag"
+	"os"
+)
+
+const (
+	// postgres connstring that works with docker-compose
+	defaultPostgresConn = "postgres://storj:storj-pass@test-postgres/teststorj?sslmode=disable"
+	defaultSqliteConn   = "sqlite3://file::memory:?mode=memory&cache=shared"
+)
+
+var (
+	testPostgres = flag.String("postgres-test-db", os.Getenv("STORJ_POSTGRES_TEST"), "PostgreSQL test database connection string")
```

Comment:
This flag is somewhat problematic, it would end up in the final binary. Not sure how to solve this though.